### ISSUE TITLE
Skip login not working

### ISSUE THREAD

**Issue by rmgpinto** [kubernetes/dashboard#4023]
##### Environment

```
Installation method: Amazon EKS
Kubernetes version: 1.13
Dashboard version: kubernetesui/dashboard:v2.0.0-beta1
Operating system: Amazon EKS
```

##### Steps to reproduce

I've added `--enable-skip-login, --disable-settings-authorizer` to the default dashboard yaml files.

##### Observed result

When clicking the `Skip` button, the page only sends a request to `https://<URL>/api/v1/login/status` and gets a `200 OK`.

##### Expected result

Skip login and redirect to kubernetes dashboard

**Comment by floreks** [comment]
Will take a look.

**Comment by dacodas** [comment]
I am also experiencing this.

**Comment by KadenLNelson** [comment]
I've experienced this as well...

**Comment by ricadelic** [comment]
for what it's worth, if you use the alternative yaml file, you don't need the skip button and you can use the dashboard.

**Comment by floreks** [comment]
Generating admin token is also super easy and does not require skip option.

**Comment by KadenLNelson** [comment]
This is definitely a luxury feature, mostly used where security is not a big regard and also where users don't want to generate an admin token (ex./ minikube).

**Comment by KadenLNelson** [comment]
> for what it's worth, if you use the alternative yaml file, you don't need the skip button and you can use the dashboard.

What do you mean?

**Comment by whillas** [comment]
Hi,
Hate to be a naysayer but the deployment
https://github.com/kubernetes/dashboard/blob/master/aio/deploy/alternative.yaml
removes the login but also means that the service account it uses (`kubernetes-dashboard`?) doesn't have permission to see anything :(

**Comment by floreks** [comment]
@whillas As expected. It is described in the docs how alternative deployment works and how it should be used.

**Comment by sguidos** [comment]
Manually cloning the repo and adding the --enable-skip-login argument to the alternative.yaml file within the container spec of the kubernetes-dashboard Deployment give me exactly what I, and I expect many others here, are looking for:  a way to easily install a fully functional dashboard on our local **development** machines without any need to authenticate or login.

Is there a way to pass the enable-skip-login args option to the installation command "kubectl create -f .../alternative.yaml" ?  That would easily give us the desired result of getting a developer-friendly dashboard installation directly from the repository, thanks.

**Comment by floreks** [comment]
Addin `--enable-skip-login` by default does not make much sense as SA used by dashboard has almost 0 privileges. If you want to alter default configurations I'd suggest to use the helm chart. We have recently added support for it.

**Comment by sguidos** [comment]
I have tried to use the Helm Chart, which has its own issues correctly using the extraArgs, see the issue I just opened, thanks.

You are correct about 0 privileges with just --enable-skip-login.  I forgot to include that I also use the `--disable-settings-authorizer `option to give me what I want, a developer-friendly local Dashboard that lets me administer my local development Kubernetes.

Also, once I find out how to pass these options to the Hem Chart, it would be nice to have the alternative.yaml exposed as a separate Helm Chart installation.
